cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillRunnerService.java

```diff
@@ -446,31 +445,12 @@ private void createKubeResourceQuota(String namespace, V1ResourceQuota resourceQ
   }
 
   /**
-   * Copy volumes into the new namespace for deployments created via the KubeTwillRunnerService
+   * Copy secret volumes into the new namespace for deployments created via the KubeTwillRunnerService
    * TODO: (CDAP-18956) improve this logic to be for each pipeline run
    */
-  private void copyVolumes(String namespace, String cdapNamespace) throws IOException {
+  private void copySecretVolumes(String namespace, String cdapNamespace) throws IOException {
```

Comment:
we shouldn't be copying these either, though as you mentioned in your comment we can do this as a follow up once the overall approach is verified.